src/site/content/en/fast/chrome-ux-report-api/index.md

```diff
@@ -0,0 +1,388 @@
+
+This API has been built from the ground up with the goals of providing developers with simple, fast, and comprehensive access to CrUX data. The CrUX API only reports _field-based_ user experience data, unlike the existing [PageSpeed Insights API](https://developers.google.com/speed/docs/insights/v5/get-started), which also runs _lab-based_ Lighthouse performance audits. The CrUX API is streamlined and can quickly serve user experience data, making it ideally suited for real-time auditing applications.
+
+To ensure that developers have access to all of the metrics that matter most—[Largest Contentful Paint](https://web.dev/lcp/) (LCP), [First Input Delay](https://web.dev/fid/) (FID), and [Cumulative Layout Shift](https://web.dev/cls/) (CLS)—the CrUX API audits and monitors these [Core Web Vitals](https://web.dev/vitals/#core-web-vitals) at both the origin and URL level.
+
+So let's dive in and see how to use it!
+
+## Querying origin data
+
+Origins in the CrUX dataset encompass all underlying page-level experiences. The example below demonstrates how to query the CrUX API for an origin's user experience data using cURL on the command line.
+
+```bash/0,3
+API_KEY="[YOUR_API_KEY]"
+curl "https://chromeuxreport.googleapis.com/v1/records:queryRecord?key=$API_KEY" \
+  --header 'Content-Type: application/json' \
+  --data '{"origin": "https://web.dev"}'
+```
+
+Run this query interactively in the [CrUX API explorer](https://developers.google.com/web/tools/chrome-user-experience-report/api/reference/rest/v1/records/queryRecord?apix=true&apix_params=%7B%22resource%22%3A%7B%22origin%22%3A%22https%3A%2F%2Fwww.google.com%22%7D%7D).
+
+{% Aside %}
+Note that all API requests must provide a value for the `key` parameter, the placeholder for which is left as `YOUR_API_KEY`. Get your own private CrUX API key at the click of a button in the official [CrUX API documentation](https://developers.google.com/web/tools/chrome-user-experience-report/api/guides/getting-started#APIKey). For convenience, the interactive [CrUX API explorer](https://developers.google.com/web/tools/chrome-user-experience-report/api/reference/rest/v1/records/queryRecord?apix=true) does not require an API key.
+{% endAside %}
+
+The `curl` command is made up of three parts:
+
+1. The URL endpoint of the API, including the caller's private API key.
+2. The `Content-Type: application/json` header, indicating that the request body contains JSON.
+3. The JSON-encoded [request body](https://developers.google.com/web/tools/chrome-user-experience-report/api/reference/rest/v1/records/queryRecord#request-body), specifying the `https://web.dev` origin.
+
+To do the same thing in JavaScript, use the <a name="crux-api-util">`CrUXApiUtil`</a> utility, which makes the API call and returns the decoded response.
+
+```js/2
+const CrUXApiUtil = {};
```

Comment:
hmmmm now this isn't the same as https://developers.google.com/web/tools/chrome-user-experience-report/api/guides/getting-started#javascript

i like some of these changes.. maybe we can sort it out in the forthcoming PR to the crux repo? and then sync up both these other uses?

Reply:
Yes, good call. Pinged you offline to share my ideas for deduplicating these docs.